modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -38,6 +38,10 @@ The app monitoring dashboards help you answer questions about your integrated ap
 * What impact do high traffic peaks have on the message fail rate?
 * What impact do high traffic peaks of other apps running on the same server have on the performance of a given app?
 
+
+[NOTE]
+After the worker is restarted, the CPU and memory information is not available in the dashboards. To retain these statistics, use the `dashboardStats` API. Alternatively, you can use Anypoint Monitoring with a Titanium subscription to store this data.
```

Comment:
Will it be clear to your audience (or does it even matter) whether this is restarted/current information that is not available or historic/cumulative information? 
In the context of the previous text, focused on information available from the dashboards, I wonder if we should mention that before the condition of restarting a worker (a corner case), maybe using parallel wording instead of a Note?
```suggestion
The app monitoring dashboards cannot provide CPU and memory information if the worker is restarted. In this case, you either retain this information by using the `dashboardStats` API or by storing it using Anypoint Monitoring with a Titanium subscription.
```